addons/google_calendar/models/google_calendar.py

```diff
@@ -1,22 +1,20 @@
 # -*- coding: utf-8 -*-
 
+from datetime import datetime, timedelta
+import logging
 import operator
 import simplejson
 import urllib2
 
+from dateutil import parser
+import pytz
+
 import openerp
 from openerp import tools
-from openerp import SUPERUSER_ID
 from openerp.tools import DEFAULT_SERVER_DATE_FORMAT, DEFAULT_SERVER_DATETIME_FORMAT, exception_to_unicode
```

Comment:
arrange your import <a href="https://github.com/odoo/odoo/blob/b9977ef5d30d98017b9f2d96bf151511b2eeafd2/doc/reference/guidelines.rst#imports">properly</a> and use openerp.fields.Datetime/Date classes' to_string and from_string methods instead DEFAULT_SERVER_*_FORMAT